# Construction of GSM for *Vibrio natriegens* and production of beta-carotene

## 1. Introduction

### 1.1 Literature review of the cell factory (*V. natriegens*)

The fast-growing marine *Vibrio natriegens* is an emerging synthetic biology chassis and a promising candidate for an industrial cell factory. The bacterium possesses many desirable qualities for a cell factory making it interesting for industrial application. This includes a high growth and substrate uptake rate, metabolic diversity, genetic tractability, and tolerance against high salinity (Hoff et al. 2020). These properties could remarkably decrease production expenses and thus increase profits. *V. natriegens* belongs to the class of gammaproteobacteria, just like *E.coli*, and provides a very promising alternative to this commonly used industrial workhorse. 

The high growth rate of *V. natriegens* makes it highly interesting as a commercial cell factory. *V. natriegens* is reported to have the highest growth rate hitherto observed in a non-pathogenetic organism and has a generation time of less than 10 minutes (Hoffart et al. 2017). In comparison, the growth rate for *E. coli* is usually 15-20 minutes. This trait could prove very timesaving and speed up the Design, Build, Test, and Learn (DBTL) cycle for the cell factory development (Eichmann et al. 2019).

The metabolic diversity and requirement of high salinity of *V. natriegens* offers a remarkable potential to decrease the production costs. In industrial fermentation, the cost of water is a significant expense. *V. natriegens* is ubiquitous in the marine environment and thus thrive in high salinity. This means that seawater can be used in the fermentation process for *V. natriegens* and thus significantly reducing the expense of water. Furthermore, the marine bacterium has a high metabolic diversity and can utilize a wide range of substrates including cheap substrates such as sucrose and glycerol (Hoffart et al. 2017; Ellis et al. 2019). Using *V. natriegens* as a cell factory could thus prove very economically favorable.

There are currently several established molecular biology techniques for *V. natriegens* including electroporation, conjugation, and CRIPSPR-Cas9 (Hoff et al. 2020). Recent studies have revealed that many methods and tools for production in *E. coli* can be applied in *V. natriegens* (Tschirhart et al. 2019; Eichmann et al. 2019). The codon usage in the two organisms are fairly similar (Hoff et al. 2020). A recent study investigated how well genetic components commonly used for *E. coli* work in *V. natriegens* (Tschirhart et al. 2019). This revealed that many of the synthetic biology tools can also be applied in the fast-growing vibrio. It has been suggested that *E. coli* tools and protocols could be a good starting point for developing more tools for *V. natriegens* (Hoff et al. 2020). This signifies that tools and methods for *V. natriegens* can hopefully be expeditiously developed for the optimization of *V. natriegens* as a cell factory.


### 1.2 Literature review of the compound (beta-carotene)

The natural product beta carotene is terpenoid that are naturally produced in fungi, plant and fruits. It is a colored red-orange pigment by which is it often used for food coloring in the food and beverage industry. It is also used for natural cosmetic products and besides being a pigment, it is also the most common source for provitamin A. The production of vitamin A is of interesting because vitamin A has multiple functions in the body, such as vision and the immune system. To express beta carotene in *V. natriegens*, the beta carotene pathway from Vibrio campbellii and the mevalonate (MVA) biosynthetic pathway from the Gram-positive bacterium *Lactobacillus acidophilus* were introduced to get the right precursors. 
Three pathways are relevant for beta carotene production (Figure 1). *Vibrio natriegens* already produce the important precursors isopentenyl pyrophosphate (IPP) and dimethylallyl pyrophosphate (DMAPP) through its upstream methylerythritol 4-phosphate (MEP) pathway, but this product seems to be a bottleneck in the production of beta carotene. To solve this issue, the MVA pathway from *Lactobacillus acidophilus* is introduced and thereby more IPP is produced. IPP is then used in the beta carotene pathway from *Vibrio campbellii* to reach the final product. Beta carotene have already been experimentally heterologously expressed in *V. natriegens* and other microbial chassis with great success. By making a computed *V. natriegens* cell factory the goal is to increase the production rate of beta-carotene by model optimizing (Ellis et al. 2019).

![alt text](BetaCarotene-pathways.png "Pathway")

*Figure 1*: The relevant pathways for beta-carotene production. *Vibrio natriegens* already contains the MEP-pathway, but the MVA and beta carotene pathways must be added. Figure from Ellis et al. 2019.

The market value of beta carotene has increased with the raised awareness of its properties (applications?) and its health benefits. During the past years there have been an increased demand from the consumers for natural and organic products, which is why many companies have started using beta carotene as ingredients in their products. The global market size of beta carotene was 314.14 billion US dollars in 2020 and the market size is estimated to grow to 380.37 billion US dollars before 2025. In Europe, the most preferred dye in food processing is beta carotene and it is also becoming the most preferred dye in the American market. Furthermore, the Asian market are big contributors to the global beta carotene market due to the increased use of cosmetics. At the moment, the beta carotene industry is dominated by the two companies DSM N.V. and BASF SE. but they are followed by the Danish company Chr. Hansen. The increasing market value as a result of the raising demand seems to keep growing during the next many years. This makes the production of beta carotene interesting and finding a new better production organism can therefore be essential for future beta carotene production. (https://www.marketdataforecast.com/market-reports/beta-carotene-market)

## 2. Problem definition

The market for beta-carotene will increase the coming years and therefore a more efficient way of producing the compound can be necessary. By introducing the heterologous pathway for beta-carotene production into the bacterium *V. natriegens* one would have an efficient and fast-growing organism as a cell factory for beta-carotene production.

Today beta-carotene is mainly produced chemically which can be damaging to the environment. It would therefore have an effect on the climate change mitigation to change a larger part of the chemical produced beta-carotene to bioproduced beta-carotene. One of the reasons it is interesting to introduce the beta-carotene pathway into *V. natriegens* instead of using for example *Yarrowia lipolytica* is that *V. natriegens* has shown good results being cultivated in sea water. This will reduce the use of drinking water significantly and as clean drinking water is a limiting resource it can help the UN with sustainable development goal 6.

## 3. Reconstruction of a new GSM for your cell factory host of interest

In [6]:
from cobra.io import read_sbml_model

V. natriegens ATCC 14048's metabolism was investigated by Long et al. by carbon-13 flux analysis in order to determine intracellular metabolic fluxes of this organism. These analysis were performed in a minimal media(M9) with glucose as an carbon source. It is for which reason why this project has chosen V. natriegens ATCC 14048's refseq from ncbi to work with in order to have as much simularity as possible to the isolate used.

In [2]:
!carve --refseq GCF_001456255.1 --output Vibrio-natriegens-ATCC_14048-M9.xml --gapfill M9 --init M9

Running diamond for the first time, please wait while we build the internal database...
diamond v2.0.4.142 (C) Max Planck Society for the Advancement of Science
Documentation, support and updates available at http://www.diamondsearch.org

#CPU threads: 32
Scoring parameters: (Matrix=BLOSUM62 Lambda=0.267 K=0.041 Penalties=11/1)
Database input file: /usr/local/lib/python3.6/dist-packages/carveme/data/input/bigg_proteins.faa
Opening the database file... Permission denied
 [0.002s]
Error: Error opening file /usr/local/lib/python3.6/dist-packages/carveme/data/generated/bigg_proteins.dmnd
Failed to run diamond (wrong arguments).
File exists, skipping.
diamond v2.0.4.142 (C) Max Planck Society for the Advancement of Science
Documentation, support and updates available at http://www.diamondsearch.org

#CPU threads: 32
Scoring parameters: (Matrix=BLOSUM62 Lambda=0.267 K=0.041 Penalties=11/1)
Temporary directory: 
Opening the database... No such file or directory
 [0s]
Error: Error opening file

In [50]:
modelM9 = read_sbml_model('Vibrio-natriegens-ATCC_14048-M9.xml')

Now, just for some basic information about the model of Vibrio natriegens created from carveme:

In [51]:
print("Number of reactions:", len(modelM9.reactions))
print("Number of metabolites:", len(modelM9.metabolites))
print("Number of genes included in the model:", len(modelM9.genes))
print("% of genes in model compared to the number of coding genes in Vibrio natrigens(4451 genes):",(len(modelM9.genes)/4451)*100)


Number of reactions: 2473
Number of metabolites: 1569
Number of genes included in the model: 1383
% of genes in model compared to the number of coding genes in Vibrio natrigens(4451 genes): 31.071669287800496


Comparing the two models´ growth rate with the actually growth rate of V. natriegens experimentially determined in minimal- and complex media:

In [52]:
print("Maximal growth rate of Vibrio natriegens in M9 medium", modelM9.optimize().objective_value)

Maximal growth rate of Vibrio natriegens in M9 medium 0.8175477356286588


The specific growth rate of V. natrigens grown as a batch culture in minimal media(M9 media) was found to be 1.70 1/h (a doubling time of app. 24 min)(Long et al. (2017)).
As can se above the specific growth rate for this project's model has a growth rate of 0.817 1/h. Hence, some difference is present between the actual specific growth rate of V. natriegens and this project's model.
This could be due to uptake rates of carbon sources and/or  intracellular fluxes not being optimal compared to what the true values found by Long et al.

Long et al. further found, using carbon-13 flux analysis, that V. natriegens had an glucose rate uptake of 21.4 mmol/g_DW/h 

In [53]:
solution = modelM9.optimize()
solution.fluxes.EX_glc__D_e

-10.0

Currently, the glucose uptake of the model is 10 mmol/g/h, which has to be changed in order to make the model of V. natriegens resemble reality.
For that a copy of modelM9 is made:

In [54]:
modelc = modelM9.copy()

In [55]:
modelc.reactions.EX_glc__D_e.bounds = -21.4,-21.4
(modelc.optimize().objective_value)

0.9222026623715347

In [56]:
with modelc:
    
    modelc.reactions.EX_glc__D_e.bounds = -1000,-1000
    print(modelc.optimize())
    solution2 = modelc.optimize()
    print(solution2.fluxes.EX_glc__D_e)
    


<Solution 0.922 at 0x7f2ebf38ed30>
-1000.0


**it does not seem to affect the specifc growth more by chaning the flux of glucose uptake to higher values...**

So the specific growth rate has been improved to be 0.922 1/h
But it does still not correspond to the actual data found by Long et al.
For further optimization of the specific growth rate of this project's model the uptake rate of oxygen was changed;
Long et. al. found that V. natriegens had an oxygen uptake rate of 28 mmol/g/h.
This project's model on minimal M9 media of V. natriegens has a oxygen uptake rate of 10 mmol/g/h:

In [57]:
solution.fluxes.EX_o2_e

-10.0

Hence, the oxygen uptake rate was changed to 28 mmol/g/h

In [58]:
modelc.reactions.EX_o2_e.bounds = -28, -28
print(modelc.optimize().objective_value)
solution=modelc.optimize()
print(solution.fluxes.EX_o2_e)

0.9222026623715811
-28.0


No change in the specific growth rate is seen when changing the bounds of oxygen uptake

Other observations made by Long et. al. 13-C metabolic flux analysis was a high glycolytic flux for certain enzymes:
80 ± 0.5 for phosphoglucose isomerase,
169 ± 1 for glyceraldehyde 3-phosphate dehydrogenase,
a flux of 27 ± 1  via phosphoenolpyruvate carboxylase,
17 ± 1 for citrate synthase,
7 ± 0.4 for α-ketoglutarate dehydrogenase

Above fluxes have been normalized to
glucose uptake rate of 100(21.4 mmol/g/h)

modelc.reactions.PGI = phosphoglucose isomerase = app. 0.8*21.4 mmol/g/h
modelc.reactions.GAPD = glyceraldehyde 3-phosphate dehydrogenase = app. 1.69*21.4 mmol/g/h
modelc.reactions.PPC = Phosphoenolpyruvate carboxylase = app. 0.27*21.4 mmol/g/h
modelc.reactions.MCITS = methylated citrate synthase = app. 0.17*21.4 mmol/g/h
model.reactions.AKGDH = α-ketoglutarate dehydrogenase = app. 0.07*21.4 mmol/g/h

In [59]:
modelc.reactions.PGI.bounds = -0.8*21.4,0.8*21.4
modelc.reactions.GAPD.bounds = -1.69*21.4,1.69*21.4
modelc.reactions.PPC.bounds = -0.17*21.4,0.17*21.4
modelc.reactions.AKGDH.bounds =-0.07*21.4,0.07*21.4

In [60]:
print(modelc.optimize().objective_value)

0.9222026623715385


No affect on the specific growth rate occured when changing the above fluxes to resemble reality.
Only the change of glucose uptake rate has affected the specific growth rate in this model grown on M9.

Additionally, Long et al. found that some pathways are inactive during aerobic growth with glucose as carbon source:
The EntnerDoudoroff pathway , the glyoxylate shunt, and
gluconeogenesis via phosphoenolpyruvate carboxykinase  were all characterised as inactive:

The EntnerDoudoroff pathway is an alterntive route in prokaryotes to catabolize glucose to pyruvate; the pathway results in a net yield of 1 ATP, 1 NADH, and 1 NADPH. Opposed the glycolysis has a net yield of 2 ATP and 2 NADH. 
Hence, it could be hypothesized that by inactivating The EntnerDoudoroff pathway a higher flow through the glycolysis would occur resulting in a higher specific growth rate. 
6-phosphogluconate dehydratase and 2-keto-3-deoxyphosphogluconate aldolase are two of the key enzymes in the EntnerDoudoroff pathway(Conway et al. 1982) and are therefor deactivated:

In [61]:
modelc.reactions.EDA.bounds = 0,0 #deactivation of 2-keto-3-deoxyphosphogluconate aldolase also known as 
#2-dehydro-3-deoxy-phosphogluconate aldolase in KEGG

In [62]:
modelc.reactions.EDD.bounds = 0,0  #deactivation of 6-phosphogluconate dehydratase

In [63]:
print(modelc.optimize().objective_value)

0.9222026623715385


The specific growth rate only increase by -2.5202063e-14...
now the other two pathways are deactivated(the glyoxylate shunt, and
gluconeogenesis via phosphoenolpyruvate carboxykinase)

In [64]:
modelc.reactions.PPCK.bounds = 0,0 #deactivaring phosphoenolpyruvate carboxykinase
modelc.reactions.ICL.bounds = 0,0 #deactivating the glyoxylate shunt

In [65]:
modelc.optimize().objective_value

0.922202662371536

Visualizing the core metabolism of *V. natriegens* using an existing map of *E. coli*

In [66]:
import escher

In [67]:
solution = modelc.optimize()

In [68]:
escher.Builder('e_coli_core.Core metabolism',
               reaction_data=solution.fluxes.to_dict()).display_in_notebook()

The model is saved as an xml file.

In [69]:
from cobra.io import write_sbml_model
write_sbml_model(modelc, "modelc.xml")

To validate the model of *V. natriegens(ATCC 14048)* a memote report was created:

In [23]:
%%time
!memote report snapshot --skip test_stoichiometric_consistency \
    --skip test_find_metabolites_not_produced_with_open_bounds \
    --skip test_find_metabolites_not_consumed_with_open_bounds Vibrio-natriegens-ATCC_14048-M9.xml --filename Vibrio-natriegens-ATCC_14048-M9.html

============================= test session starts ==============================
platform linux -- Python 3.6.12, pytest-6.1.2, py-1.9.0, pluggy-0.13.1
rootdir: /usr/local/lib/python3.6/dist-packages/memote/suite/tests
collected 146 items / 1 skipped / 145 selected                                 

../../../../usr/local/lib/python3.6/dist-packages/memote/suite/tests/test_annotation.py F [  0%]
FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF..         [ 44%]
../../../../usr/local/lib/python3.6/dist-packages/memote/suite/tests/test_basic.py . [ 45%]
.....F..........F...FF                                                   [ 60%]
../../../../usr/local/lib/python3.6/dist-packages/memote/suite/tests/test_biomass.py . [ 60%]
F.....F.F                                                                [ 67%]
../../../../usr/local/lib/python3.6/dist-packages/memote/suite/tests/test_consistency.py s [ 67%]
FFssssssssssssssss.F^C
CPU times: user 7.16 s, sys: 1.28 s, total: 8.44 s
Wall 

**Add text that describes memote reporten!!!!**

In [24]:
%%time
!memote report snapshot --skip test_stoichiometric_consistency \
    --skip test_find_metabolites_not_produced_with_open_bounds \
    --skip test_find_metabolites_not_consumed_with_open_bounds modelc.xml --filename modelc.html

UnboundLocalError: local variable 'child' referenced before assignment

## 4. Computer-Aided Cell Factory Engineering

The heterologous pathway for beta carotene production will be inserted into the constructed *V. natriegens* model as proof-of-concept to assess the model and *V. natriegens* as an emerging cell factory. For *V. natriegens* to produce beta carotene, 9 reaction steps must be added to the model. The required reaction steps are identified using KEGG PATHWAY maps of *V. natriegens* and *V. campbellii*. The first reaction is the conversion of farnesyl disphosphate and isopentenyl diphosphate into geranylgeranyl diphosphate and diphosphate. The stochiometry of the reaction is defined:

**Defining and adding reaction 0**

In [4]:
from cobra import Reaction, Metabolite

Reaction_0_beta=Reaction('Step0_beta') # Farnesyl diphosphate + isopentenyl diphosphate <=> geranylgeranyl diphosphate + diphosphate
geranylgeranyl_PP=Metabolite(id='Geranylgeranyl_pp_c', compartment='c')
Reaction_0_beta.add_metabolites({modelc.metabolites.frdp_c: -1, # Farnesyl diphosphate
                                 modelc.metabolites.ipdp_c: -1, # Isopentenyl diphosphate
                                 geranylgeranyl_PP: 1, # Geranylgeranyl diphosphate
                                 modelc.metabolites.ppi_c: 1 # diphosphate
                                })

modelc.add_reactions([Reaction_0_beta]) # Adding reaction to model

NameError: name 'modelc' is not defined

This procedure is repeated for the remaining reaction steps in the conversion of farnesyl diphosphate into beta carotene. This has been done in file Beta+MVA-pathway.ipynb, and a model with the beta carotene pathway incorporated (modelBC) has been saved.

In [7]:
modelBC = read_sbml_model('modelBC.xml')

OSError: [Errno The file with 'filename' does not exist, or is not an SBML string. Provide the path to an existing SBML file or a valid SBML string representation: 
%s] modelBC.xml

In order to have flux, a boundary is implemented for the last metabolite, beta carotene.

In [80]:
modelc.add_boundary(modelBC.metabolites.Beta_carotene_c, type='demand')

<Reaction DM_Beta_carotene_c at 0x7f2ebf3edcf8>

**Predicting maximal theoretical beta carotene production**

After the heterogous pathway have been added to the model, the theoretical maximal yield of beta carotene is calculated. This is done by setting the final reaction, yielding beta carotene, as the model objective and thus maximize the flux through that reactionin a flux balance analysis . The maximal theoretical yield of beta carotene will then be the objective value of the solution. The growth rate,  beta carotene production and yield on glucose is calculated to evaluate the heterologous expression of beta carotene in the constructed GSM.

In [81]:
with modelBC:
    modelBC.objective = modelBC.reactions.Growth    
    print('Growth rate for V. natriegens [h^-1]:', modelBC.optimize().objective_value)
    modelBC.objective = modelBC.reactions.Step8_beta
    beta_carotene_production = modelBC.optimize().objective_value
    print('Max. beta-carotene production [mmol gDW^-1 h^-1]:', beta_carotene_production)
    print('Theoretical maximum yield [mmol_beta / mmol_glc]:', beta_carotene_production / (-1*modelBC.reactions.EX_glc__D_e.flux))

Growth rate for V. natriegens [h^-1]: 0.922202662371536
Max. beta-carotene production [mmol gDW^-1 h^-1]: 2.0183670931059052
Theoretical maximum yield [mmol_beta / mmol_glc]: 0.09431621930401426


This gives a result of the maximal theoretical production of beta carotene being 2.02 mmol gDW<sup>-1</sup>h<sup>-1</sup> in *V. natriegens*. In order to compare this result, the beta-carotene pathway reactions are also inserted into the *E. coli* iML1515 model (See file Betacarotene_in_Ecoli.ipynb). This resulted in a maximum theoretical production of 0.94 mmol gDW<sup>-1</sup>h<sup>-1</sup>. Production in the constructed *V. natriegens* model is thus more than 2-fold higher than with the *E. coli* iML1515 model. The maximal growth rate of *V. natriegens* is also slightly higher than *E. coli* iML1515. The maximal growth rate of *V. natriegens* ended up being 0.92 h<sup>-1</sup>, and for *E. coli* iML1515 the maximal growth rate is 0.88 h<sup>-1</sup>. Furthermore, the yield of beta carotene in *V. natriegens* and *E. coli* are more or less the same on 0.094 mmol<sub>beta</sub>/mmol<sub>glc</sub> with the first difference on the fourth significant decimal.

**Adding precursor**

In an attemp to increase beta carotene production, the MVA-pathway has furthermore been implemented in the model. The MVA pathway produces ornathate which is a precursor for beta carotene. The implementation of the MVA pathway has been done in Beta+MVA-pathway.ipynb, and modelBCMVA was saved and contains both the beta carotene and MVA pathway.

In [ ]:
modelBCMVA = read_sbml_model('modelBCMVA.xml')

## 5. Discussion

Some discussion of the construction of the GSM

Beta-carotene has previously been succesfully expressed in *V. natriegens* and thus verifies that the heterologous expression is possible (Ellis et al. 2019). In that study, the titer of beta-carotene was approximately 0.5 mg/L without adding the MVA pathway. The beta-carotene production in *V.natriegens* was increased approximately 6-fold in LBv2 rich medium when the MVA pathway from from *Lactobacillus acidophilus* NCFM was additionally introduced (Ellis et al. 2019). When the MVA-pathway was introduced into *V. natriegens*, the titer was almost 3 mg/L (Ellis et al. 2019). 

The maximal theoretical production of betacaroten was more than 2-fold higher in *V. natriegens* compared to the *E. coli* iML1515 model. 

The heterologous expression of beta-carotene in *V. natriegens* in this study was meant as a proof-of-concept to investigate if production in this host was possible. The aim was thus not to improve the yield of beta-carotene. For this reason, it might be possible to experimentally achieve a higher yield.

## 6. Conclusion

In this work, a genome-scale model of *V. natriegens* has been constructed. The GSM was validated with experimental data from the literature to improve the model and its predictions. Achieving the experimental growth rate in the GSM has not been possible so far. The constructed GSM predicted a growth rate of 0.92 h<sup>-1</sup> while an
experimental study reported a growth rate of 1.70 h<sup>-1</sup>.  The model was evaluated using memote which showed that the constructed GSM **is so and so good**. As a proof-of-concept, a heterologous pathway for beta-carotene was inserted in the model. Beta-carotene has previously been heterologously expressed in *V. natreigens*, and expression was also succesful in the constructed GSM. The yield of beta carotene in *V. natriegens* is almost the same as *E. coli* at 0.094 mmol<sub>beta</sub>/mmol<sub>glc</sub>. The GSM predicted a maximal theoretical production of 2.02 mmol gDW<sup>-1</sup>h<sup>-1</sup> beta carotene in *V. natriegens* and thus 2-fold higher than the maximal theoretical production in the *E. coli* iML1515 model. This indicates that *V. natriegens* could be a favorable host for heterologous expression of beta carotene.

## References

Conway, T. (1992). The Entner-Doudoroff pathway: history, physiology and molecular biology. FEMS Microbiol. Lett. 103, 1–28. doi:10.1111/j.1574-6968.1992.tb05822.x.

Eichmann, J., Oberpaul, M., Weidner, T., Gerlach, D., and Czermak, P. (2019). Selection of High Producers From Combinatorial Libraries for the Production of Recombinant Proteins in Escherichia coli and Vibrio natriegens. Front. Bioeng. Biotechnol. 7. doi:10.3389/fbioe.2019.00254.

Ellis, G. A., Tschirhart, T., Spangler, J., Walper, S. A., Medintz, I. L., and Vora, G. J. (2019). Exploiting the Feedstock Flexibility of the Emergent Synthetic Biology Chassis Vibrio natriegens for Engineered Natural Product Production. Mar. Drugs 17, 679. doi:10.3390/md17120679.

Hoff, J., Daniel, B., Stukenberg, D., Thuronyi, B. W., Waldminghaus, T., and Fritz, G. (2020). Vibrio natriegens : an ultrafast‐growing marine bacterium as emerging synthetic biology chassis. Environ. Microbiol. 22, 4394–4408. doi:10.1111/1462-2920.15128.

Hoffart, E., Grenz, S., Lange, J., Nitschel, R., Müller, F., Schwentner, A., et al. (2017). High substrate uptake rates empower Vibrio natriegens as production host for industrial biotechnology. Appl. Environ. Microbiol. 83. doi:10.1128/AEM.01614-17.

Long, C. P., Gonzalez, J. E., Cipolla, R. M., and Antoniewicz, M. R. (2017). Metabolism of the fast-growing bacterium Vibrio natriegens elucidated by 13 C metabolic flux analysis. Metab. Eng. 44, 191–197. doi:10.1016/j.ymben.2017.10.008.

Tschirhart, T., Shukla, V., Kelly, E. E., Schultzhaus, Z., Newringeisen, E., Erickson, J. S., et al. (2019). Synthetic Biology Tools for the Fast-Growing Marine Bacterium Vibrio natriegens. ACS Synth. Biol. 8, 2069–2079. doi:10.1021/acssynbio.9b00176.
